### DISCUSSION TITLE
Support for deploymentsettings when using pac pipeline deploy

### DISCUSSION THREAD

**Discussion by nitkavd** [microsoft/powerplatform-build-tools#1100] [Ideas]
The pac solution import supports passing the settings file to setup environment variables and connections (--settings-file).
Some tenants are configured to run with the Power Platform pipelines, and all deploys should be done using these pipelines, so we cannot use the pac solution import approach.
It seems that the pipeline deploy UI (and the pipelines framework) support the use of deploymentsettings. When entering the configuration in the UI upon deploy, the settings seem to be persisted in a deploymentsettings field of the pipeline run. Passing the deploymentsettings is not yet available in the CLI.

Can we expect this addition? Is there a workaround, while still using pac pipeline deploy?

**Comment by sanjeev40084** [comment]
this would be really nice from the governance side. without this the connections in target env. get saved as user who is deploying the solution for the first time. We want to be able to add connection in target env as service account, share with service principal and provide that information as deployment settings json file to pipeline so when user deploys the pipeline for the first time, it picks up the connection information provided from json file rather than creating new under user who is running the pipeline.
